diagnostic.md

```diff
@@ -51,13 +61,19 @@ Write the `curl` request you'd use to delete a single donut, using whatever ID
 you'd like.
 
 ```sh
-# your answer here
+curl --request DELETE http://www.example.com/donuts/2
 ```
 
 Write an AJAX request to delete a single donut from the server.
 
 ```js
-let deleteDonut = /* your answer here */;
+let deleteDonut = function() {
+  $.ajax({
+    url: 'http://www.example.com/donuts/2',
+    type: 'DELETE'
+  })
+};
+
 ```
```

Comment:
it's good to add your .done and .fail as well. Always write them, consider them key to writing any ajax request.